roles/core/conman/molecule/default/verify.yml

```diff
@@ -0,0 +1,124 @@
+---
+- name: Verify
+  hosts: all
+  tasks:
+
+    - name: Collect package facts for RedHat
+      package_facts:
+        manager: auto
+      when:
+        - ansible_facts.os_family == "RedHat"
```

Comment:
For OpenSUSE, the collect currently fails because the package `python2-rpm` is missing from the container. I will update the image whenever possible. In the meantime, a workaround would be to install `python2-rpm` in prepare.yml for OpenSUSE OS family.
This would allow to collect the packages for OpenSUSE as well, and ensure the packages are installed.